java-checks/src/main/java/org/sonar/java/checks/StringLiteralInsideEqualsCheck.java

```diff
@@ -67,4 +74,22 @@ private static boolean isEquals(IdentifierTree tree) {
       "equalsIgnoreCase".equals(tree.name());
   }
 
+  private JavaQuickFix computeQuickFix(MethodInvocationTree tree) {
+    String equalsParameterValue = QuickFixHelper.contentForTree(tree.arguments().get(0), context);
```

Comment:
I would extract `tree.arguments().get(0)` into `equalsArgument` because it is repeated later in the method and also because it becomes a bit more explicit.